lib/ansible/modules/system/service.py

```diff
@@ -530,7 +530,8 @@ def sysv_is_enabled(name):
             return True
         elif out.startswith('disabled'):
             return False
-        elif sysv_exists(service_name):
+        # `systemctl is-enabled` returns rc 0 for enabled, 1 for disabled, and 4 for service file not found
+        elif rc > 1 and sysv_exists(service_name): 
```

Comment:
this has changed over time, rc is not a reliable way to judge existence ... looking at alternative way .. currently dbus seems like only reliable way to get the info.